File: JavaScript/Practice/LeetCode/Easy/Maximum Product of Two Elements in an Array.js
// https://leetcode.com/problems/maximum-product-of-two-elements-in-an-array/description/

/**
 * @param {number[]} nums
 * @return {number}
 */
var maxProduct = function(nums) {
    nums.sort((a, b) => a - b);
    return (nums[nums.length - 2] - 1) * (nums[nums.length - 1] - 1);
};

File: C#/Practice/LeetCode/Easy/Maximum Product of Two Elements in an Array.cs
// https://leetcode.com/problems/maximum-product-of-two-elements-in-an-array/description/

public class Solution {
    public int MaxProduct(int[] nums) {
        Array.Sort(nums);
        return (nums[nums.Length - 2] - 1) * (nums[nums.Length - 1] - 1); 
    }
}